File: RaceAppJava/Model/src/main/java/org/model/Participant.java
package org.model;

import com.google.gson.annotations.SerializedName;

public class Participant extends Entity<Long>{
    String nume;
    @SerializedName("capMotor")
    int capMotor;
    String echipa;
    String cnp;

    public Participant(Long id, String nume, int capMotor, String echipa, String cnp) {
        super.id = id;
        this.nume = nume;
        this.capMotor = capMotor;
        this.echipa = echipa;
        this.cnp = cnp;
    }

    public String getNume() {
        return nume;
    }

    public void setNume(String nume) {
        this.nume = nume;
    }

    public int getCapMotor() {
        return capMotor;
    }

    public void setCapMotor(int capMotor) {
        this.capMotor = capMotor;
    }

    public String getEchipa() {
        return echipa;
    }

    public void setEchipa(String echipa) {
        this.echipa = echipa;
    }

    public String getCnp() {
        return cnp;
    }

    public void setCnp(String cnp) {
        this.cnp = cnp;
    }

    @Override
    public String toString() {
        return "Participant{" +
                "nume='" + nume + '\'' +
                ", capMotor=" + capMotor +
                ", echipa=" + echipa +
                ", cnp='" + cnp + '\'' +
                '}';
    }
}

File: RaceAppC#/model/Participant.cs
namespace model
{
    public class Participant : Entity<long>
    {
        public string Nume {  get; set; }
        public int CapMotor {  get; set; }
        public string Echipa { get; set; }
        public string Cnp {  get; set; }

        public Participant(long id, string Nume, int CapMotor, string Echipa, string Cnp)
        {
            this.id = id;
            this.Nume = Nume;
            this.CapMotor = CapMotor;
            this.Echipa = Echipa;
            this.Cnp = Cnp;
        }
    }
}
